lib/queries/get-challenges-by-tag.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
```

Comment:
I have replied in #52. 

I definitely set out to make this an abstract function, but it didn't feel great, hence how it turned out..